config/dev.exs

```diff
@@ -51,9 +51,9 @@ config :phoenix, :stacktrace_depth, 20
 
 # Configure your database
 config :cadet, Cadet.Repo,
-  username: "postgres",
-  password: "postgres",
-  database: "cadet_dev",
+  username: "###",
+  password: "###",
+  database: "###",
```

Comment:
Please don't change this

Reply:
Can we instead put this file into .gitignore?

Reply:
If you need to have different credentials, just override it in `dev.secrets.exs` in the exact same manner.